pytorch_lightning/metrics/functional/classification.py

```diff
@@ -116,6 +138,23 @@ def stat_scores_multiple_classes(pred: torch.Tensor, target: torch.Tensor,
 def accuracy(pred: torch.Tensor, target: torch.Tensor,
              num_classes: Optional[int] = None,
              reduction='elementwise_mean') -> torch.Tensor:
+    '''
```

Comment:
```suggestion
    """
```
for consistency with the rest of the docs